app/src/main/java/in/tosc/digitaloceanapp/activities/DropletActivity.java

```diff
@@ -40,10 +41,6 @@
 import retrofit2.Callback;
 import retrofit2.Response;
 
-/**
```

Comment:
Should we be removing the javadocs that give credit to whoever created the class though, even if it's boilerplate from Android Studio?

Reply:
We could update them to all follow a standard date format and have more information regarding the class so it's not _just_ the creator information.

Reply:
```java
/**
 * The {@link AppCompatActivity} class responsible for displaying {@link Droplet} information.
 *
 * @author the-dagger on 11/26/2016
 */
```

Maybe something like this?

Reply:
@Alcha yes as I have only removed some stuff that was not needed so i believe there won't be any issue, even if there is, the conflicts for such PRs could be resolved easily. 
About removing the creator comments, they are quite inconsistent as the name mentioned there might not coincide with our real name or our github username and also just giving credit to the one who created that file and not to the ones who maintain and enhance it is not correct in my opinion.

Reply:
Then we can change it to show who created it and then who now maintains it. Especially if the majority of the code remaining is still written by the original author.

Reply:
```java
/**
 * The {@link AppCompatActivity} class responsible for displaying {@link Droplet} information.
 *
 * @author created by the-dagger on 11/26/2016, maintained by AkshayCHD
 */
```

Reply:
@Alcha currently the project is small but still we have alot  of people who contributed in the past and there would be many who would be contributing in the future. I see no use of maintaining this record when all the information about each and eery contribution is avaliable on github.

Reply:
Fair enough, it's a difference of opinion then.

In that case, we'll have to leave it to @the-dagger and @championswimmer to decide.

Reply:
Yaa, it is better to leave it to them